config/initializers/titleize_exceptions.rb

```diff
@@ -0,0 +1,14 @@
+class String
```

Comment:
I suggest not to monkeypatch the class string, I think that this could be just a regular rails helper

Reply:
👍

Reply:
Fixed in 9199d80